optuna/multi_objective/trial.py

```diff
@@ -382,6 +378,9 @@ def __hash__(self) -> int:
 
     # TODO(ohta): Implement `__repr__` method.
 
+    def _get_values(self) -> List[Optional[float]]:
+        return self.values
```

Comment:
Do you still think we should keep this method? I know it's private and a small detail.